web/docs/handbook/contributing.mdx

```diff
@@ -39,4 +39,5 @@ One of the Supabase maintainer will approve your PR in the next few hours.
 <!-- Add your github handle below this line. Every name should be on a new line -->
 [awalias](https://github.com/awalias).
 [kiwicopple](https://github.com/kiwicopple).
-<!-- Add your github above this line. Every name should be on a new line -->
\ No newline at end of file
+<!-- Add your github above this line. Every name should be on a new line -->
+[abhinav-TB](https://github.com/abhinav-TB).
```

Comment:
Hey @abhinav-TB 

Nice job! 

Perhaps you can move your name above this line?

`<!-- Add your github above this line. Every name should be on a new line -->`

Reply:
@kiwicopple done

Reply:
🔥 thanks for contributing